WordPress/Classes/ViewRelated/Post/PrivateSiteURLProtocol.h

```diff
@@ -20,8 +20,4 @@
 + (void)registerPrivateSiteURLProtocol;
 + (void)unregisterPrivateSiteURLProtocol;
 
-+ (nonnull NSURLRequest *)requestForPrivateSiteFromURL:(nonnull NSURL *)url;
```

Comment:
I think that we the removal of the uses of this methods we no longer need this class at all. 
You probably can remove it completely.

Reply:
Agree!  Nice catch!